### ISSUE TITLE
Unable to install ifcPlugins and BinarySerializers plugins

### ISSUE THREAD

**Issue by azOn2018** [opensourceBIM/BIMserver#891]
I'm having problem installing couple of essential plugins (ifcPlugins and BinarySerializers) in BIMserver v1.5.118. The issue #861 may have some relevance to this but this seems like a bit different error. Please note, I'm running this version of BIMserver (a local git repository) from within Eclipse. Also I was able to install the BIMserver with some other plugins either as -plugins in Arguments for LocalDevBimServerStarter or from Maven. This is a clean installation after previously using v1.5.103. The error I'm getting are shown below:
 
![ifcplugin-installation-error](https://user-images.githubusercontent.com/40860221/50035745-ce6e2800-fffb-11e8-85fc-d5cf1cb09d27.PNG)
![serializer-issue](https://user-images.githubusercontent.com/40860221/50035746-ce6e2800-fffb-11e8-9c4b-47f859ac1bbe.PNG)

I did another test where I downloaded BIMserver v1.5.117 jar file and successfully installed it with all the essential plugins including the two I'm having problem with. For this I just ran the jar file and this is not from within Eclipse. 

Any suggestion on how to fix this issue? I've already spent a lot of time trying various things but unfortunately I'm stuck with this frustrating issue.

**Comment by rubendel** [comment]
These are simply not "binary" compatible. The plugins interface has undergone some changes. When running BIMserver from GIT, your best guess is to also run the plugins you need from GIT. Only after a 1.5.118 release of BIMserver has been done will the plugins also be released.

**Comment by azOn2018** [comment]
Thanks Ruben. This is what I tried initially i.e. install from Git but after it didn't work, I also attempted to install these plugins from Maven. After several failed attempts I posted this issue here. Anyway, the good news is after several more attempts, I finally managed to install ifcPlugins and BinarySerializers from Git. Not sure why it worked in the end as to be quite honest I didn't try anything new.
